doc/input-output-reference/src/overview/group-operational-faults.tex

```diff
@@ -353,9 +363,11 @@ \subsubsection{Inputs}\label{inputs-7-015}
 
 This field provides the name of a Thermostat Offset Fault object that causes the humidistat offset fault. It should be the name of a FaultModel:ThermostatOffset object. This field is required for the Type ThermostatOffsetDependent. This is used as a multiplier to the reference humidistat offset value. This schedule should be set to a non-zero value when a fault is applicable and ``0.0'' when it is not. If this field is blank, the schedule has values of 1.0 for all time periods.
 
-\subsection{An example of IDF codes for the thermostat/humidistat offset fault models:}\label{an-example-of-idf-codes-for-the-thermostathumidistat-offset-fault-models}
+\paragraph{IDF Examples}\label{an-example-of-idf-codes-for-the-thermostathumidistat-offset-fault-models}
```

Comment:
The idf example that come prior to FaultModel:ThermostatOffset are misaligned.  Would be good to clean them up here.  All of the other fault model idf examples look good.